app/src/main/java/com/reminisense/fa/activities/BarcodeScanner.java

```diff
@@ -0,0 +1,175 @@
+package com.reminisense.fa.activities;
+
+import android.content.DialogInterface;
+import android.content.pm.ActivityInfo;
+import android.hardware.Camera;
+import android.os.Bundle;
+import android.os.Handler;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.KeyEvent;
+import android.view.View;
+import android.widget.Button;
+import android.widget.FrameLayout;
+
+import com.reminisense.fa.R;
+
+import net.sourceforge.zbar.Config;
+import net.sourceforge.zbar.Image;
+import net.sourceforge.zbar.ImageScanner;
+import net.sourceforge.zbar.Symbol;
+import net.sourceforge.zbar.SymbolSet;
+
+public class BarcodeScanner extends AppCompatActivity {
```

Comment:
It's good practice to add javadoc documentation here. Add a description for what this activity is for and what it does. Also, please change the file name to follow our naming convention. (i.e.*Activity.java)